aardwolf-rocket/Cargo.toml

```diff
@@ -48,7 +48,7 @@ version = "0.4.0-rc.1"
 
 [dependencies.rocket_contrib]
 version = "0.4.0-rc.1"
-features = ["tera_templates", "json"]
```

Comment:
Does ructe not count as a "feature" then?
Just curious

Reply:
no, because Rocket doesn't have any knowledge of ructe

Reply:
aha!  That makes sense :)
Guess I need to read up on "features"